orttraining/orttraining/python/training/ortmodule/_fallback.py

```diff
@@ -206,7 +206,7 @@ def fallback(self, model: torch.nn.Module, log_level: _logger.LogLevel, *inputs,
 
         assert self.is_pending(), '`fallback` can only be called when there is a pending fallback'
 
-        if log_level <= _logger.LogLevel.WARNING:
+        if log_level <= _logger.LogLevel.INFO:
```

Comment:
If the log level is `WARNING`, then this log statement will not print. I think we should keep it the way it was so that it prints when the log level is `WARNING` (which is the default).